client/my-sites/people/people-invites/index.jsx

```diff
@@ -194,7 +194,7 @@ class PeopleInvites extends React.PureComponent {
 
 		const dialogButtons = [
 			<Button busy={ deleting } primary onClick={ this.handleClearAll }>
-				{ translate( 'Clear All' ) }
+				{ translate( 'Clear all' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Clear All Accepted' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/button-sentence-case)